protobufs/livekit_analytics.proto

```diff
@@ -121,8 +121,9 @@ enum AnalyticsEventType {
   SIP_CALL_INCOMING = 37;
   SIP_CALL_STARTED = 38;
   SIP_CALL_ENDED = 39;
+  REPORT = 40;
```

Comment:
Adding an `AnalyticsEvent` to report different kinds of data to analytics. Open to a better name.